sourmash/commands.py

```diff
@@ -13,6 +13,7 @@
 from . import MinHash, load_sbt_index, create_sbt_index
 from . import signature as sig
 from . import sourmash_args
+from sourmash.sourmash_args import load_file_as_signatures
```

Comment:
`sourmash_args` is already loaded, see line above - you can use `sourmash_args.load_file_as_signatures`. I think we should pick a single way to refer to `load_file_as_signatures` throughout this file - fine with the way you're doing it, you just need to change the other calls in the file 😉 

(Generally, within a package, I try to avoid importing the top level package and prefer the `from .` notation. Here, you would be able to use `from .sourmash_args import load_file_as_signatures`)

Reply:
thanks - missed those. fixing!